src/style.css

```diff
@@ -1,3 +1,67 @@
+html {
+  font-family: Roboto, sans-serif;
```

Comment:
Consider adding fallback font families to improve robustness of your font settings. For example: `font-family: Roboto, 'Arial', sans-serif;`